### ISSUE TITLE
Table of cloud resources

### ISSUE THREAD

**Issue by ACharbonneau** [datacarpentry/cloud-genomics#38]
The workshop now ends with an exercise in determining what resources fit various projects. 

We want a reference table that will allow students/instructors to compare various cloud resources without having to memorize all of them. 

I have pros/cons written in paragraph format for HPCCs and headers for a number of others at: https://github.com/datacarpentry/cloud-genomics/blob/gh-pages/_episodes/04-which-cloud.md

It can either all be in paragraph/bullet format like I have. Or mine can be re-imagined in whatever format works.

**Comment by dvanic** [comment]
I can't see the table, or paragraph?.. Are you on this, Amanda?

**Comment by ACharbonneau** [comment]
The only paragraph I have done and merged in the HPCC one. 

Info for XSEDE  and AWS is PR https://github.com/datacarpentry/cloud-genomics/pull/46 

I was thinking paragraphs with more in depth info, and then a table to summarise, but that can get done after the paragraphs. 

I know literally nothing about Google Cloud, Azure, JetStream, CyVerse or Atmosphere...So if I do them, I'm going to have to spend some time learning what they are...it would be great if someone who actually uses those services or knows about them could work on those bits.

**Comment by tracykteal** [comment]
Agree, we should find people who know about other options. @ctb @ljcohen maybe could help with JetStream. @devbioinfoguy with ACI-REF. @JasonJWilliamsNY with CyVerse.

**Comment by ErinBecker** [comment]
@ACharbonneau and @tracykteal - we were planning on pushing the "publish" button on these lessons today to have it out before the holiday. Is this something that can wait for the next release or can we get it into this release today?

**Comment by ACharbonneau** [comment]
Answering here for multiple issues: I'm fine with dropping content for now at any placeholders, but I'm teaching now, and until at least 7pm, so I can't do the drops. If someone can drop them, I'll go back through and make issues to put them back in later.

**Comment by amanda-tan** [comment]
Hello, all -- is there still interest in expanding this to other clouds? I have extensive experience with both Azure and GCP and would be willing to write guides to replicating an AWS setup on both those cloud providers.

**Comment by ACharbonneau** [comment]
Hey @amanda-tan this issue is about adding high-level overviews of the pluses/minuses of each option. So, for instance, why would I choose Azure over AWS. It would be *great* if you could add in useful information about the ones you're familiar with. The page for that is here: https://github.com/datacarpentry/cloud-genomics/blob/gh-pages/_episodes/04-which-cloud.md

We have a separate page where we low-key host information for logging on to non-AWS cloud resources. It's not part of the main lesson, but the lesson does link to it. If you want to add info there, that's useful too: https://github.com/datacarpentry/cloud-genomics/blob/gh-pages/_extras/LaunchingInstances.md

We'd appreciate either, but helping to flesh out the cloud options in Episode 4 would be much, much more immediately useful, as that's part of every workshop. The LaunchingInstances page is nice to have, but never gets used in lessons, and is only meant as a resource for users after a workshop, so we're not sure whether anyone ever looks at it.
